ui/apps/platform/src/services/AccessScopesService.ts

```diff
@@ -4,14 +4,8 @@ import { Traits } from '../types/traits.proto';
 
 const accessScopessUrl = '/v1/simpleaccessscopes';
 
-/*
- * TODO: ROX-13585 - remove the pre-postgres constants once the migration to postgres
- * is completed and the support for BoltDB, RocksDB and Bleve is dropped.
- */
 export const defaultAccessScopeIds = {
-    Unrestricted: 'io.stackrox.authz.accessscope.unrestricted',
     UnrestrictedPostgres: 'ffffffff-ffff-fff4-f5ff-ffffffffffff',
```

Comment:
Would it make sense to rename the `...Postgres` constants to `Unrestricted` and `DenyAll` ?

Reply:
Yes, I agree.